contracts/contracts/PoolDeposits.sol

```diff
@@ -184,16 +191,14 @@ contract PoolDeposits {
     aaveLendingContract.deposit(address(daiContract), amount, 30);
 
     timeJoined[msg.sender] = now;
-    depositedDai[msg.sender] = amount;
+    depositedDai[msg.sender] = depositedDai[msg.sender].add(amount);
     totalDepositedDai = totalDepositedDai.add(amount);
   }
 
   /// @dev Internal function completing the actual redemption from Aave and sendinding funds back to user
-  function _withdrawFunds() internal {
-    uint256 amount = depositedDai[msg.sender];
-    _removeEmergencyVote();
```

Comment:
Removing emergancy vote only happens on `exit`.
There is a rule that you cannot partially remove deposit if you have voted to 'declare an emergancy'.
Since this (`_withdrawFunds`) is shared by both `exit` and `withdrawDeposit` it was removed.